lib/signs/utilities/audio.ex

```diff
@@ -402,4 +402,25 @@ defmodule Signs.Utilities.Audio do
       end)
     )
   end
+
+  @spec send_pa_message(PaMessages.PaMessage.t(), %{integer() => DateTime.t()}, function()) :: %{
+          integer() => DateTime.t()
+        }
+  def send_pa_message(pa_message, pa_message_plays, send_audio_fn) do
+    case Map.get(pa_message_plays, pa_message.id) do
+      nil ->
+        send_audio_fn.()
+
+      last_sent ->
+        if DateTime.diff(DateTime.utc_now(), last_sent, :millisecond) >= pa_message.interval_in_ms do
+          send_audio_fn.()
+        else
+          Logger.warn("pa_message: action=skipped id=#{pa_message.id}")
+        end
+    end
+    |> then(fn
+      :ok -> Map.put(pa_message_plays, pa_message.id, DateTime.utc_now())
```

Comment:
`PaEss.Updater.play_message` doesn't currently return `:ok` for the new SCU code path, so it should be changed to do that. Alternatively, we could just remove the branching here, since I don't think we expect a recoverable failure case anyway.